packages/tests/server/validators.test.ts

```diff
@@ -221,6 +223,57 @@ test('myzod', async () => {
   await close();
 });
 
+test('arktype schema', async () => {
```

Comment:
```suggestion
test('arktype schema - [not officially supported]', async () => {
```

maybe just a note that these aren't something we've explicitely added support for? wording is hard though ;)

Reply:
Fine by me 😄